app.js

```diff
@@ -65,44 +65,42 @@ adminAppRouter.get('/', async (req, res) => {
 });
 
 /**
- * @summary Get APIs
+ * @summary Get account indexes
  */
 appRouter.get(`/${api}`, async (req, res) => {
   try {
-    const MAX_PAGE_SIZE = 500;
-    const { page } = req.query;
-    /**
-     * Return 400 errors if page[size]/page[number] are not valid
-     */
-    if (page) {
-      const { size, number } = page;
-      const isInvalidSize = (size !== '') && (size <= 0 || size > MAX_PAGE_SIZE);
-      const isInvalidNumber = (number !== '') && number <= 0;
-      const errors = [];
-
-      if (isInvalidSize || isInvalidNumber) {
-        if (isInvalidSize) errors.push(`page[size] should an integer between 1 to ${MAX_PAGE_SIZE}.`);
-        if (isInvalidNumber) errors.push('page[number] should an integer starts at 1.');
-        return res.status(400).send(badRequest(errors));
+    const { q, organizationCode } = req.query;
+    if (!q && !organizationCode) {
+      res.status(400).send(badRequest(['At least one parameter is required.']));
+    } else {
+      // If there's additional parameters in the query object, oracle will complain
+      // because it doesn't recognize them. Thus, we only add the ones we need,
+      // and the ones that are defined.
+      // TODO: There's got to be a less ugly way to do this.
+      const params = {};
+      if (q) {
+        params.q = q;
+      }
+      if (organizationCode) {
+        params.organizationCode = organizationCode;
```

Comment:
Good catch, I should also add some params cleaner function to the skeleton. You could probably use [lodash](https://lodash.com/docs/4.17.10) to shorten the code:

```suggestion
      _.forEach(req.query, (value, key) => {
        if (value !== '') params[key] = value;
      });
```

you will need to require lodash at the top of the file:

```js
const _ = require('lodash');
```

Reply:
Mentioned this in hipchat, but I still need a way to only include parameters that I'm expecting (q and organizationCode). For now, I'll continue explicitly getting the expected parameters. In the future, this should be inherited from the swagger definition.